### ISSUE TITLE
sage-10.4 - sagemath_setup not valid sagemath subpackage

### ISSUE THREAD

**Issue by strogdon** [cschwan/sage-on-gentoo#788]
While emerging `sage_setup-9999`:

```
 * ERROR: sci-mathematics/sage_setup-9999::sage-on-gentoo failed (unpack phase):
 *   sage_setup is not a valid sagemath subpackage
 * 
 * Call stack:
 *     ebuild.sh, line  136:  Called src_unpack
 *   environment, line 4473:  Called sage-git_src_unpack
 *   environment, line 4429:  Called die
 * The specific snippet of code:
 *       [[ -d "${EGIT_CHECKOUT_DIR}/pkgs/${SAGE_PKG}" ]] || die "${SAGE_PKG} is not a valid sagemath subpackage";
```

**Comment by kiwifb** [comment]
I see, I did not check that bit. Can you check the other ones?

**Comment by kiwifb** [comment]
OK only "_" ebuilds seem to be impacted, which sage_docbuild too but not any other.

**Comment by kiwifb** [comment]
Should be fixed by 8460e4ff278f9c1932981589d0c212229b8537b7 - give time for the overlay to synchronize.

**Comment by strogdon** [comment]
Fixed.
